apps/web/src/components/sidebar/NestedSafesPopover/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { SvgIcon, Popover, Button, Box } from '@mui/material'
+import { useContext } from 'react'
+import type { ReactElement } from 'react'
+
+import AddIcon from '@/public/images/common/add.svg'
+import { ModalDialogTitle } from '@/components/common/ModalDialog'
+import { CreateNestedSafe } from '@/components/tx-flow/flows/CreateNestedSafe'
+import { TxModalContext } from '@/components/tx-flow'
+import { NestedSafesList } from '@/components/sidebar/NestedSafesList'
+import { NestedSafeInfo } from '@/components/sidebar/NestedSafeInfo'
+import Track from '@/components/common/Track'
+import { NESTED_SAFE_EVENTS } from '@/services/analytics/events/nested-safes'
+
+export function NestedSafesPopover({
+  anchorEl,
+  onClose,
+  nestedSafes,
+}: {
+  anchorEl: HTMLElement | null
+  onClose: () => void
+  nestedSafes: Array<string>
+}): ReactElement {
+  const { setTxFlow } = useContext(TxModalContext)
+
+  const onAdd = () => {
+    setTxFlow(<CreateNestedSafe />)
+    onClose()
+  }
+
+  return (
+    <Popover
+      open={!!anchorEl}
+      anchorEl={anchorEl}
+      onClose={onClose}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'left',
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'left',
+      }}
+      slotProps={{ paper: { sx: { width: '300px' } } }}
+    >
+      <ModalDialogTitle
+        hideChainIndicator
+        onClose={onClose}
+        sx={{ borderBottom: ({ palette }) => `1px solid ${palette.border.light}` }}
+      >
+        Nested Safes
+      </ModalDialogTitle>
+      <Box
+        sx={{
+          p: 3,
+          pt: 2,
+          display: 'flex',
+          flexDirection: 'column',
+          maxHeight: '590px',
+        }}
+      >
```

Comment:
Hinweise: Box and Typography take sx props directly (which may look a bit neater).

```suggestion
      <Box
        p={3}
        pt={2}
        display="flex"
        flexDirection="column"
        maxHeight="590px"
      >
```

Though in this particular case, the Stack component would be even better.